### PR TITLE
CI for pre-release

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Add single test for pre-release packages, shall prevent our issue with TM

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
this will require update required tests...

File: .github/workflows/ci_test-full.yml [hunk 1]
```diff
@@ -18,11 +18,17 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-18.04, windows-2019, macOS-10.15]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.8, 3.9]
```
Comment (ananthsub) [inline .github/workflows/ci_test-full.yml:21]: why's 3.7 removed?

Comment (Borda) [reply .github/workflows/ci_test-full.yml:21]: our test started to be quite large and I do not expect that there is anything on pure python that would work on both versions around 3.6 and 3.8 and was not working in 3.7

Comment (Borda) [reply .github/workflows/ci_test-full.yml:21]: @carmocca suggestion?

Comment (carmocca) [reply .github/workflows/ci_test-full.yml:21]: Sounds good to me.

Lets remove Python 3.6 support in 8 months (its end-of-life)

https://endoflife.date/python

**Review by carmocca — APPROVED** [review]

File: .github/workflows/ci_test-full.yml (hunk 1 above)
Comment (Borda) [reply .github/workflows/ci_test-full.yml:21]: ol, then we set 3.7 as min, that is a good point...
well py.36 is one of the most used python version for us...
https://pypistats.org/packages/pytorch-lightning

**Review by awaelchli — APPROVED** [review]

**Review by ananthsub — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

**Pull request merged**